.github/workflows/sign-and-release-to-github.yml

```
@@ -64,17 +64,32 @@ jobs:
       - name: "Lint"
         run: npm run lint
 
-      - name: "Configure add-on for stage site"
+      - name: "Configure add-on for the stage site"
         run: npm run config:stage
+        if: github.ref_type == 'branch'
 
-      - name: "Set YYYY.MM.DD.minutes version"
+      - name: "Configure add-on for the production site"
+        run: npm run config:stage
```

suggestion: I think this should be for prod. 

```suggestion
        run: npm run config:prod
```

---

Ooof good catch :sweat_smile: Also why I didn't update the docs yet: want to try it out properly with the next tag, because there's probably more mistakes. 073bda7